tests/ClientTest.php

```diff
@@ -395,23 +389,6 @@ public function convertExceptionDataProvider()
                     ],
                 ],
             ],
-            [
-                new \RuntimeException('foo'),
-                [
-                    'auto_log_stacks' => false,
```

Comment:
You're no longer testing for this case, why?

Reply:
Explained above, for exceptions we always want a stacktrace.